opentrace.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=OpenTrace
+Keywords=traceroute;OpenTrace;network;
+Exec=OpenTrace
+Comment=OpenTrace is a cross-platform GUI wrapper for NextTrace. Bringing you a sense of familiarity.
+Icon=/app/share/icons/hicolor/1024x1024/apps/opentrace.png
```

Comment:
1024px icon is not supported, and drop the full path.

Reply:
Oops I'm sorry! This should be /scalable/, or let me put it in /512x512/ later

Reply:
no. it should be the icon name here, nothing else. no path. no extension.

Reply:
Thank you! fixed